pkgs/by-name/zu/zulip/package.nix

```diff
@@ -1,24 +1,25 @@
 { lib
 , fetchFromGitHub
 , buildNpmPackage
-, electron_32
+, python3
+, electron
```

Comment:
We need to pin a specific Electron major version: https://github.com/NixOS/nixpkgs/pull/279545#discussion_r1445127278

Yes, `electron_35` is what I’m waiting for. I got a couple steps in to packaging that myself but I haven’t been successful yet, and I imagine the Electron nixpkgs maintainers will be able to do it more quickly.

Reply:
The `electron_35` PR is now merged into `master`.